FSharp-4.0/StaticMethodArgumentsDesignAndSpec.md

```diff
@@ -46,9 +46,9 @@ From the point of view of the type provider author, using the modified ProvideTy
       let m = ProvidedMethod("ExampleMethWithStaticParam", [ ], typeof<int>, IsStaticMethod = false)
       m.DefineStaticParameters(staticParams, (fun nm args ->
           let arg = args.[0] :?> int
-          let parms = [ for i in 1 .. arg -> ProvidedParameter("arg" + string i, typeof<int>)]
+          let params = [ for i in 1 .. arg -> ProvidedParameter("arg" + string i, typeof<int>)]
```

Comment:
`params` is a reserved keyword in F#

Reply:
@dsyme: did you want me to change this to `params_` or back to `parms`?

Reply:
Thanks - I updated to `parameters` just now via direct commit